src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java

```diff
@@ -174,22 +172,18 @@ void GIVEN_user_component_config_with_small_periodic_interval_WHEN_interval_elap
         assertEquals(1, putLogEventsRequests.size());
         for (PutLogEventsRequest request : putLogEventsRequests) {
             assertEquals(calculateLogStreamName(THING_NAME, LOCAL_DEPLOYMENT_GROUP_NAME), request.logStreamName());
-            assertEquals("/aws/greengrass/UserComponent/" + AWS_REGION + "/UserComponentA",
-                    request.logGroupName());
+            assertEquals("/aws/greengrass/UserComponent/" + AWS_REGION + "/UserComponentA", request.logGroupName());
             assertNotNull(request.logEvents());
-            assertEquals(50, request.logEvents().size());
-            assertEquals(51200, request.logEvents().stream().mapToLong(value -> value.message().length())
-                    .sum());
+            assertEquals(60, request.logEvents().size());
+            assertEquals(61440, request.logEvents().stream().mapToLong(value -> value.message().length()).sum());
```

Comment:
why is the size and number of events changed?

Reply:
Because the active log file is also included